qa/workunits/cephtool/test.sh

```diff
@@ -1514,6 +1514,9 @@ function test_mon_osd_pool_set()
   wait_for_clean
   ceph osd pool set $TEST_POOL_GETSET pgp_num 10
 
+  ceph osd pool set $TEST_POOL_GETSET pg_num 106
```

Comment:
could you squash this commit into the previous one? or add a prefix ("qa/workunits/cephtool/test.sh: ") to the title of the commit message of this commit?